src/vm/docs/mod.rs

```diff
@@ -302,10 +302,12 @@ const EQUALS_API: SpecialAPI = SpecialAPI {
     input_type: "A, A, ...",
     output_type: "bool",
     signature: "(is-eq v1 v2...)",
-    description: "Compares the inputted values, returning `true` if they are all equal. Note that _unlike_ the `(and ...)` function, `(is-eq ...)` will _not_ short-circuit.",
+    description: "Compares the inputted values, returning `true` if they are all equal. Note that 
+_unlike_ the `(and ...)` function, `(is-eq ...)` will _not_ short-circuit. All values supplied to
+is-eq _must_ be the same type.",
     example: "(is-eq 1 1) ;; Returns true
-(is-eq 1 false) ;; Returns false
-(is-eq \"abc\" 234 234) ;; Returns false
+(is-eq true false) ;; Returns false
+(is-eq \"abc\" 234 234) ;; Throws type error
```

Comment:
`(is-eq "abc" 234 234)` is actually an _analysis error_, reported as:

    expecting expression of type 'int', found '(buff 3)'